spec/lib/vmdb/settings_spec.rb

```diff
@@ -202,4 +214,25 @@
 
     include_examples "password handling"
   end
+
+  describe ".for_resource" do
+    let(:server) { FactoryGirl.create(:miq_server) }
+
+    it "without database changes" do
+      settings = Vmdb::Settings.for_resource(server)
+      expect(settings.api.token_ttl).to eq "10.minutes"
+    end
+
+    it "with database changes" do
+      server.settings_changes.create!(:key => "/api/token_ttl", :value => "2.minutes")
+      settings = Vmdb::Settings.for_resource(server)
+      expect(settings.api.token_ttl).to eq "2.minutes"
```

Comment:
I think the latter. We didn't have any tests that tested the value returned by the whole config stack, just what was in the `settings_changes` table, so we added these in addition to the array ones for completeness.